### DISCUSSION TITLE
Neo-tree.nvim source (integration)

### DISCUSSION THREAD

**Discussion by prncss-xyz** [zk-org/zk-nvim#67] [Share]
Just wanted to let you know I have written a [source plugin](https://github.com/prncss-xyz/neo-tree-zk.nvim) for [neo-tree.nvim](https://github.com/nvim-neo-tree/neo-tree.nvim). So far I'm finding it useful in my workflow. If there is anything you think should be done differently please let me know. And thanks for zk, enjoying it a lot!

**Comment by mickael-menu** [comment]
This is pretty cool, thanks for sharing! I'll add it to the README.

This discussion could be relevant to: https://github.com/mickael-menu/zk/discussions/125

**Reply by prncss-xyz** [reply]
Great, I've opened a feature request about that!

**Comment by riodelphino** [comment]
@prncss-xyz Thank you for `neo-tree-zk.nvim` plugin !

I've created a PR for upstream syncing based on it.
[zk-org/neo-tree-zk.nvim PR#2](https://github.com/zk-org/neo-tree-zk.nvim/pull/2)

And now I also extended Snacks.explorer for zk, with reference to `neo-tree-zk.nvim`.
[discussions #264](https://github.com/zk-org/zk-nvim/discussions/264)

Thanks !
